src/plugins/authentication/http-basic-authentication-plugin.ts

```diff
@@ -1,86 +1,64 @@
-import type { GraphileConfig } from 'graphile-config'
+import type { GraphileConfig } from "graphile-config";
 import {
-    AUTHENTICATED_CALLER_PROPERTY,
-    AUTHENTICATION_FEATURE_LABEL,
-} from './constants.js'
-
-declare global {
```

Comment:
Doesn't removing this have the affect that `basicHttpAuthenticationAdminCredentials` is going to be added to ExampleOptions regardless of whether this plugin is imported/used? I was hoping to demonstrate how plugins can intelligently add config only if they're imported

Reply:
Yes, I missed that this was meant to be an entrypoint, my mistake.